PURL-SPECIFICATION.rst

```diff
@@ -377,7 +377,7 @@ To parse a `purl` string in its components:
     - The `value` is the percent-decoded right side
     - UTF-8-decode the `value` if needed in your programming language
     - Discard any key/value pairs where the value is empty
-    - If the `key` is `checksums`, split the `value` on ',' to create
+    - If the `key` is `checksum`, split the `value` on ',' to create
       a list of `checksums`
```

Comment:
@williamboman One question: on line 381, should `checksums` be changed to remove the backticks, as you did on line 324?